pages/common/go-run.md

```diff
@@ -0,0 +1,9 @@
+# go run
+
+> Compile and run Go code.
+> Does not leave a binary on disk.
```

Comment:
The fact that this is technically very wrong **really** annoys me.

Reply:
Have you got a suggestion?

Reply:
@pixelcmtd do you mean because it still has to save the executable somewhere? In that case it's probably removed right after being used, so it does indeed leave no binary.

Also, a small suggestion:
```suggestion
> Does not leave a binary on the disk.
```